docs/user_guide/backend_configuration.rst

```diff
@@ -0,0 +1,286 @@
+       'Zstd': hdf5plugin._filters.Zstd}
+
+    And likewise for ``AVAILABLE_ZARR_COMPRESSION_METHODS``.
+
+We can confirm these values are saved by re-printing that particular dataset configuration...
+
+.. code-block:: python
+
+    print(dataset_configuration)
+
+.. code-block:: bash
+
+    acquisition/MyTimeSeries/data
+    -----------------------------
+      dtype : float64
+      full shape of source array : (3,)
+      full size of source array : 0.00 GB
+
+      buffer shape : (2,)
+      expected RAM usage : 0.00 GB
+
+      chunk shape : (1,)
+      disk space usage per chunk : 0.00 MB
+
+      compression method : Zstd
+      compression options : {'clevel': 3}
+
+
+Interfaces and Converters
+-------------------------
+
+The normal workflow when writing an NWB file using a ``DataInterface`` or ``NWBConverter`` is simple to configure.
+
+The following example uses the :ref:`example data <example_data>` available from the testing repo...
+
+.. code-block:: python
+
+    from datetime import datetime
+
+    from dateutil import tz
+    from neuroconv import ConverterPipe
+    from neuroconv.datainterfaces import SpikeGLXRecordingInterface, PhySortingInterface
+    from neuroconv.tools.nwb_helpers import (
+        make_or_load_nwbfile,
+        get_default_backend_configuration,
+        configure_backend,
+    )
+
+    # Instantiate interfaces and converter
+    ap_interface = SpikeGLXRecordingInterface(
+        file_path=".../spikeglx/Noise4Sam_g0/Noise4Sam_g0_imec0/Noise4Sam_g0_t0.imec0.ap.bin"
+    )
+    phy_interface = PhySortingInterface(
+        folder_path=".../phy/phy_example_0"
+    )
+
+    data_interfaces = [ap_interface, phy_interface]
+    converter = ConverterPipe(data_interfaces=data_interfaces)
+
+    # Fetch available metadata
+    metadata = converter.get_metadata()
+
+    # Create the in-memory NWBFile object and retrieve a default configuration
+    backend="hdf5"
+
+    nwbfile = converter.create_nwbfile(metadata=metadata)
+    backend_configuration = converter.get_default_backend_configuration(
+        nwbfile=nwbfile,
+        backend=backend,
+    )
+
+    # Make any modifications to the configuration in this step, for example...
+    backend_configuration["acquisition/ElectricalSeriesAP/data"].compression_method = "Blosc"
+
+    # Configure and write the NWB file
+    nwbfile_path = "./my_nwbfile_name.nwb"
+    converter.run_conversion(
+        nwbfile_path=nwbfile_path,
+        nwbfile=nwbfile,
+        backend=backend,
+        backend_configuration=backend_configuration,
+    )
+
+.. note::
+
+    If you do not intend to make any alterations to the default configuration for the given backend type, then you can follow the classic workflow...
+
+    .. code-block::
+
+        converter = ConverterPipe(data_interfaces=data_interfaces)
+
+        # Fetch available metadata
+        metadata = converter.get_metadata()
+
+        # Create the in-memory NWBFile object and retrieve a default configuration
```

Comment:
Here,  I think this is copied from the other structure but does not apply here, right?

Reply:
What about this does not apply? This is just the bare minimum snippet to showcase the one new keyword argument for `backend : Literal["hdf5", "zarr"]` automatically configures things without custom overrides or explicit configuration passed

Reply:
Oh I see now, the phrasing of the comment